Libs/RaptureCore/src/main/java/rapture/kernel/SeriesApiImpl.java

```diff
@@ -554,6 +558,28 @@ private void removeRepoFromCache(String authority) {
             if (notEmpty.contains(uri)) continue;
             deleteSeries(context, uri);
         }
+        
+        if (recurse) {
```

Comment:
i did suggest something in my previous comment.  Just use an anonymous inner class. Same as lambda here.